vendor/BUILD

```diff
@@ -202,7 +202,6 @@ filegroup(
         "//vendor/github.com/ghodss/yaml:all-srcs",
         "//vendor/github.com/globalsign/mgo/bson:all-srcs",
         "//vendor/github.com/globalsign/mgo/internal/json:all-srcs",
-        "//vendor/github.com/go-ini/ini:all-srcs",
```

Comment:
Do we know why the existing tools did not remove this automatically?

Reply:
no idea. one does not peer into the madness that is godeps.

Reply:
LOL!